compatibility_server/compatibility_checker_server.py

```diff
@@ -194,14 +195,14 @@ def log_message(self, format, *args):
         type=int,
         default=8888,
         help='port to which the server should bind')
-    parser.add_argument(
-        '--export_metrics',
-        action='store_true',
-        help='whether or not metrics should be exported to stackdriver')
+    export_metrics = os.environ.get('EXPORT_METRICS') is not None
```

Comment:
Is `is not None` necessary here?

Reply:
Since `export_metrics` will be logged, I think it makes sense to have the flag be either `True` or `False` instead of something arbitrary for log readability.